pom.xml

```diff
@@ -68,53 +78,48 @@
     </profiles>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <version>${maven.compiler.version}</version>
+                    <configuration>
+                        <source>${java.version}</source>
+                        <target>${java.version}</target>
+                        <forceJavacCompilerUse>true</forceJavacCompilerUse>
```

Comment:
why is this needed?

Reply:
After consulting with Jay, he indicated that he had added this while attempting to get the code working with Java 11, but he believes that this setting is no longer needed. I will go ahead and remove this part of the setup, and verify that I can still build the WAR locally.